scripts/release_testing_setup.sh

```diff
@@ -22,17 +22,26 @@ if [ -f "${HOME}/.cocoapods/repos" ]; then
   find "${HOME}/.cocoapods/repos" -type d -maxdepth 1 -exec sh -c 'pod repo remove $(basename {})' \;
 fi
 
-mkdir -p "${local_sdk_repo_dir}"
-echo "git clone from github.com/firebase/firebase-ios-sdk.git to ${local_sdk_repo_dir}"
-set +x
-# Using token here to update tags later.
-git clone -q https://"${BOT_TOKEN}"@github.com/firebase/firebase-ios-sdk.git "${local_sdk_repo_dir}"
-set -x
+if [ "$TESTINGMODE" = "release_testing" ]; then
+  mkdir -p "${local_sdk_repo_dir}"
+  echo "git clone from github.com/firebase/firebase-ios-sdk.git to ${local_sdk_repo_dir}"
+  set +x
+  # Using token here to update tags later.
+  git clone -q https://"${BOT_TOKEN}"@github.com/firebase/firebase-ios-sdk.git "${local_sdk_repo_dir}"
+  set -x
+  cd  "${local_sdk_repo_dir}"
+elif [ "$TESTINGMODE" = "prerelease_testing" ]; then
+  git fetch --tags origin main
+  git checkout main
+fi
```

Comment:
The `release` workflow also uses this script, and I want to preserve its function until I get the `prerelease` workflow working, so I have some temporary logic added to do so.

I should be able to confirm upon the merge of this PR (there is one job in prerelease.yml that only runs on the merge of the PR).